server/config.go

```diff
@@ -93,6 +93,11 @@ type Config struct {
 	TickInterval typeutil.Duration `toml:"tick-interval"`
 	// ElectionInterval is the interval for etcd Raft election.
 	ElectionInterval typeutil.Duration `toml:"election-interval"`
+	// DisablePrevote is true to disable Raft Pre-Vote.
+	// If enabled, Raft runs an additional election phase
+	// to check whether it would get enough votes to win
+	// an election, thus minimizing disruptions.
+	DisablePreVote bool `json:"disable-pre-vote"`
```

Comment:
why not using PreVote directly?

Reply:
I want to enable it by default. use `PreVote` should distinguish whether the key exists in the TOML actually is false or not set it.

Reply:
so, if not, use true directly.

We must explicitly set to false if we want to disable it. 

Of course, you can add prevote=true in the example configuration.